examples/bootstrap/outputs.tf

```diff
@@ -1,24 +1,29 @@
 output "storage_account_name" {
   description = "Name of the Azure Storage Account object used for the Bootstrap."
   value       = module.bootstrap.storage_account.name
+  sensitive   = true
```

Comment:
If we pass an entire object and some attributes are provider-marked as sensitive, the 0.15 requires the entire object to be now sensitive.

It calls for adding a separate single non-sensitive module's output (one object with attributes manually copied) or multiple outputs (each attribute becomes a separate module's output).